script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_3.txt

```diff
@@ -1,88 +1,88 @@
 "...Yeah, I think that he's having a temper tantrum because he doesn't like living in a new place. He doesn't mind most things, but he has a bad habit of lashing out when he doesn't understand something."
 <Page14>
 "...Is that so? He doesn't seem like the type to get angry."
 <Page15>
-"...No, that's not true. He looks calm, but when he goes off, he really goes off. Rather often. In a snap."
+"...No, that's not true. He looks all docile, but when he goes off, he really goes off. Rather often. In a snap."//「……いや、そんなコトないんだなこれが。遠野はおとなしそうに見えるだけで、爆発する時は爆発するんスよ。わりと頻繁に。パチっと」
 "...Hah. Snap, you say?"
 <Page16>
-"...Yeah, yeah. Once he loses his temper, he'll lose all sense of control, so don't trust him, even if you're his upperclassman."
-...The two of them are whispering secretly to each other.
+"...Yeah, yeah. Once he loses his temper, he's indiscriminate. Don't trust him, even if you're his upperclassman."//「……そうそう。一度キレると見境がなくなるから、先輩もこいつを信用しちゃダメだぜ」
+...The two of them are whispering 'secretly' to each other.//　……二人はひそひそと内緒話をしている。emphasis mine, "内緒話" needs to be worked in somehow and it fits with the following lines.
 <Page17>
-"...You know what? If you want to talk in private, can you do it in the hallway? I can hear what you're saying at my desk, and it's giving me a headache, you know."
-"No way!! You heard me, Tohno!"
+"...You know what? If you want to talk in private, can you do it in the hallway? I can hear what you're saying and it's giving me a headache, you know."//「……あのさ。内緒話なら廊下でしてくれないか？　人の机でそういうコトされても、まる聞こえで頭痛いんだけど」"人の机" dropped.
+"No way!! You heard me, Tohno!?"//「なんと、聞こえていたというのか遠野!?」
 <Page18>
 ...I heard you.
 He even pretended to be talking in private, approached Senpai, and casually extended his hand to her waist, only to be struck down mercilessly.
-Rather than getting angry, I felt depressed by such a blatantly theatrical performance.
+Rather than getting angry, I was depressed by such a blatantly theatrical performance.
 <Page19>
-In the meantime, Senpai is covering her mouth with her hand out of surprise. I wonder how serious she really is.
+In the meantime, Senpai is covering her mouth with her hand in surprise. I wonder how serious she really is.
 ...She might have thought that she was really talking in private.
 <Page20>
-"Oh, that's why you're a liberal arts glasses guy. Listening to me and Senpai talking about our love affair is beyond pathetic. ...#Hmm, are you really that frustrated every day? Is your new sister really that beautiful? Can you introduce her to me next time?"
-"Arihiko, seriously, are you trying to start a fight with me?"
+"Oh, that's why you're a liberal arts four eyes. Listening to me and Senpai talk about our love affair is just beyond pathetic... #Hmm, are your days really that filled with frustration? Is your new sister reaaally that beautiful? Can you introduce her to me next time?"//「オゥ、これだから文系の眼鏡クンは。オレと先輩のラ・マンな話を盗み聞くとか哀れにも程がありますなァ……。#んー、そんなに日々欲求不満なんですかー？　新しい妹サマはそんなに美人なんですかァー？　今度紹介してくれますかー？」
```

Comment:
> オレと先輩のラ・マンな話を盗み聞くとか哀れにも程がありますなァ

**listening in on** me and Senpai talking about our love affair

Can you introduce me to her next time?